tests/app/helpers/conftest.py

```diff
@@ -226,6 +243,49 @@ def expected_footer_social_theme():
     return {**footer_context(), **social}
 
 
+def expected_footer_welsh_social_theme():
```

Comment:
Instead of having this that is a duplicate, I suggest just updating the existing one which gets passed in a language code.
```py
def expected_footer_social_theme(language_code: str):
    ons_url = ONS_URL_CY if language_code == "cy" else ONS_URL
    upstream_url = f"{ACCOUNT_SERVICE_BASE_URL_SOCIAL}/{language_code}"
```

Reply:
Fixed all this now, much shorter code, thanks!